src/react-components/tokens/RevokeTokenModal.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+import { Modal } from "../../react-components/modal/Modal";
+import styles from "./Tokens.scss";
+import styleUtils from "../styles/style-utils.scss";
+import { Button } from "../input/Button";
+import { Row } from "../layout/Row";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faExclamationTriangle } from "@fortawesome/free-solid-svg-icons/faExclamationTriangle";
+import { faTimes } from "@fortawesome/free-solid-svg-icons/faTimes";
+import { Column } from "../layout/Column";
+
+export function RevokeTokenModal({ onClose, revoke }) {
```

Comment:
To follow the naming convention, should this `revoke` be called something like `onRevoke`?

Reply:
Yes